src/generators/new/packagejson.js

```diff
@@ -78,5 +75,16 @@ module.exports = async ({
     packages.push('@herbsjs/herbs2knex')
     packages.push('pg')
   }
+  if (rest) {
+    packages.push('express@4.17.1')
+    packages.push('cors@2.8.5')
+    packages.push('@herbsjs/herbs2rest')
+  }
+  if (graphql) {
```

Comment:
I think this validations will increase fast, I believe we can do something more dynamic like

``` js
[...]

let pkgs2install = {
    mongo: ['mongodb'],
    postgres: ['@herbsjs/herbs2knex', 'pg'],
    rest: ['express@4.17.1','cors@2.8.5', '@herbsjs/herbs2rest' ],
    graphql: ['graphql@15.5.1', 'apollo-server@2.25.2', 'apollo-server-express@2.25.2', 'graphql-tools@8.1.0']
}

// I think it could exist a better implementation
const defaultOptions = (options) => {
    return {
        name = options.name ? options.name : 'herbs-project',
        description = options.description ? options.description : 'project generated by herbs CLI S2',
        author = options.author ? options.author : 'herbs CLI',
        license = options.license ? options.license : 'MIT',
        mongo = options.mongo ? options.mongo : false,
        postgres = options.postgres ? options.postgres : false,
        yarn = options.yarn ? options.yarn : false,
        graphql = options.graphql ? options.graphql : false,
        rest = options.rest ? options.rest : false
    }
}

module.exports = async ({
  template: { generate },
  parameters: {
    options
   },
   print
}){
    options = defaultOptions(options)
    const packages = [
        '@herbsjs/herbsshelf',
        'dotenv@10.0.0',
        'nodemon',
        'lodash.camelcase'
      ]
      let packages = []
      // if this options.key be [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) 
      // and this same key exists into out `pkg2install`
      for(const key of Object.key(options)){
          if(key && pkgs2install[pkgs2install])
              packages = packages.concat(pkgs2install[pkgs2install]);

      await installPkgs(yarn, packages, print)
}
```

ps: it's a pseudocode, must to be tested

Reply:
@italojs , we have already closed this PR, can you send your proposal as an issue and in another PR?

Thanks